Azure-Samples/chat-with-your-data-solution-accelerator
# fix:   Deleted files are visible on screen after clicking on delete button.

## Purpose
On click of delete button, pages refresh automatically and receives a notification file are deleted

Bug: https://dev.azure.com/CSACTOSOL/CSA%20Solutioning/_workitems/edit/5536




## How to Test

1. Navigate to Chat with your data URL
2. Click on 'Delete Data' tab
3. Select files to Delete
4. Scroll down and select multiple checkboxes to delete
5. Delete files
6. Observe files are still on the screen but received notification "files are deleted"

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```


![image](https://github.com/user-attachments/assets/24d578f4-8c72-41e7-a51e-58a660d5b5b9)
****
